profiles/schema.py

```diff
@@ -46,6 +46,56 @@
 )
 
 
+def update_profile(info, profile, profile_data):
+    youth_profile_data = profile_data.pop("youth_profile", None)
+    concepts_of_interest = profile_data.pop("concepts_of_interest", [])
```

Comment:
didn't you guys delete these from the `ProfileNode`?

Reply:
Yes, it's already merged in develop. Should I merge those changes into this PR?

Reply:
yes, please, do 🙂